### PR TITLE
Feat/#50 add users data hookup

### PR DESCRIPTION
**Pull request by ciaranschutte**

### CODE REVIEW

File: client/.storybook/webpack.config.js [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
   };
 
   config.resolve.modules.push(path.resolve(__dirname, '../'));
 
+  config.module.rules.push({
+    test: /\.(graphql|gql)$/,
+    exclude: /node_modules/,
+    loader: 'graphql-tag/loader',
+  });
+
```
Comment (hlminh2000) [inline client/.storybook/webpack.config.js]: In an ideal world, this is awesome :P but I think we want to hold off having any data tied into stories for now. 
Let's limit any data hook up to be done at application level and fed into UI components :)

Comment (ciaranschutte) [reply client/.storybook/webpack.config.js]: @hlminh2000 that makes sense. I actually just added this so Storybook webpack wouldn't complain about our .gql files. But I think I write a webpack exclude rule instead

Comment (ciaranschutte) [reply client/.storybook/webpack.config.js]: @hlminh2000 actually looks like that won't work
`addUser` imports the `mutations.gql` so still needs to load

File: client/components/pages/submission-system/modals/addUser/index.js [hunk 2]
```diff
@@ -152,13 +155,29 @@ const AddUserModal = ({}) => {
...
+      userLastName: data.lastName,
+      userEmail: data.email,
+      userRole: data.role,
+    };
+  };
+
+  const sendAddUser = validData =>
+    useMutation(INVITE_USER_MUTATION, {
+      variables: { user: createUserInput(validData) },
+    });
```
Comment (hlminh2000) [inline client/components/pages/submission-system/modals/addUser/index.js]: let's move this outside of this hook and define it in the page, fed to the component as `onSubmit` or something :) that way we don't expose anything to the story

**Comment by ciaranschutte** [comment]
@hlminh2000 @joneubank had to update react-apollo-hooks, so believe I broke create-program form. will fix it in this: https://app.zenhub.com/workspaces/argo-5c8c8637777e5f767e486217/issues/icgc-argo/argo-platform/307

File: client/components/pages/submission-system/modals/addUser/index.js [hunk 3]
```diff
@@ -40,13 +39,14 @@ const AddUserModal = ({ dismissModal }) => {
   const submitForm = async () => {
     try {
       const validData = await validateForm();
-      console.log(validData);
-      // Send data
+      const result = sendAddUser(validData);
```
Comment (hlminh2000) [inline client/components/pages/submission-system/modals/addUser/index.js]: I think we can just do `onSubmit(validData)` here and remove the `sendAddUser` function eh?

File: client/.storybook/webpack.config.js (hunk 1 above)
Comment (hlminh2000) [reply client/.storybook/webpack.config.js]: is this still needed?

**Comment by ciaranschutte** [comment]
@hlminh2000 removed graphql tag and extra func

**Pull request merged**